pwiz_tools/Skyline/Model/DdaSearch/MSAmandaSearchWrapper.cs

```diff
@@ -202,15 +204,22 @@ private void InitializeEngine(CancellationTokenSource token, string spectrumFile
             Settings.ConsideredCharges.Add(3);
             Settings.ChemicalData.UseMonoisotopicMass = true;
             Settings.ReportBothBestHitsForTD = false;
+            Settings.CombineConsideredCharges = false;
+            //Settings.WriteResultsTwice = true;
+            //Settings.ForceTargetDecoyMode = false;
+            //Console.WriteLine("\nReportBothBestHitsForTD CombineConsideredCharges WriteResultsTwice ForceTargetDecoyMode");
+            //Console.WriteLine($@"{Settings.ReportBothBestHitsForTD}       {Settings.CombineConsideredCharges}        {Settings.WriteResultsTwice}       {Settings.ForceTargetDecoyMode}");
             mzID.Settings = Settings;
             SearchEngine = new MSAmandaSearch(helper, _baseDir.DirPath, _outputParameters, Settings, token);
             SearchEngine.InitializeOutputMZ(mzID);
             Settings.LoadedProteinsAtOnce = (int) AdditionalSettings[MAX_LOADED_PROTEINS_AT_ONCE].Value;
             Settings.LoadedSpectraAtOnce = (int) AdditionalSettings[MAX_LOADED_SPECTRA_AT_ONCE].Value;
         }
     
-        public override bool Run(CancellationTokenSource tokenSource)
+        public override bool Run(CancellationTokenSource tokenSource, IProgressStatus status)
```

Comment:
This method should really be taking a CancellationToken instead of a CancellationTokenSource (since it is only interesting in asking whether the job has been cancelled and does not need the ability to request cancellation).

That would require changing MSAmandaSearch so that it also took a CancellationToken instead of CancellationTokenSource.

Reply:
I agree but since this requires a change in the MSAmanda DLLs and it's not specifically relevant to DIA-Umpire, let's leave that to a separate change set.